### PR TITLE
Started adding to team contributing document 

### PR DESCRIPTION
**Pull request by lwjohnst86**
I've started adding more to the team contributing workflow guide. Its at a baby phase right now, but will add to it over time.

Do any of you have any suggestions on what to add to the community/contribute/team/index.md file?

### CODE REVIEW

File: community/contribute/team/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Communication across team and collaborators
+
+Our general philosophy is to limit meetings and emails as much as is
+appropriate, and instead use text-based forms of communication. For
+shorter or quick questions, we use our Discord channel. For longer
+discussions, we use GitHub Issues. And for help or sharing information,
+we use blog posts that we add to the website.
+
+No matter which communication channel is used, we write in Markdown
```
Comment (K-Beicher) [inline community/contribute/team/index.md:11]: Should we except Discord here?

**Review by K-Beicher — APPROVED** [review]
Just the one comment, otherwise looks good to me.

File: community/contribute/team/index.md (hunk 1 above)
Comment (lwjohnst86) [reply community/contribute/team/index.md:11]: Well, you can use a limited set of Markdown, but not completely. I'll clarify in the text.

File: community/contribute/team/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+format.
```
Comment (lwjohnst86) [inline community/contribute/team/index.md:12]:
```suggestion
No matter which communication channel is used, we write in Markdown
format, though Discord uses a limited set of Markdown features.
```

**Pull request merged**